online-resources/learn-to-code-resources.md

```diff
@@ -8,6 +8,7 @@
 * [Resources for women arriving at or returning to IT](https://insimpleterms.blog/2017/10/13/resources-for-women-arriving-at-or-returning-to-it/)
 * [Notes to a Junior Developer from a Tech Lead](https://medium.com/@charlottebrf/notes-to-a-junior-developer-from-a-tech-lead-599e4ab7aac4)
 * [Unicornelia blog](http://unicornelia.com/coding-resources/)
+* [Checklist for Getting a Job] (https://www.notion.so/jsjoeio/Checklist-for-Getting-a-Job-1a4d673b06c048b8b1aec13d735939ad)
```

Comment:
That's a really great resource, thanks @jsjoeio 👍

Reply:
One thing I'm wondering is if we should duplicate this across the tech tests & interviews file also

Reply:
@charlottebrf Yeah, I was wondering if I put this in the right place.

What do you think? Should we add it to the others? Feel free to do it or, let me know if you want me to do it and I'll open another pull request!